wrapper/lib/zn-data.js

```diff
@@ -77,7 +77,8 @@ export function ZnData (plugin) {
       WebhookEvents: ['/webhook_events/:id'],
       Workspaces: ['/workspaces/:id'],
       WorkspaceInvitees: ['/workspaces/:workspaceId/invitees/:id'],
-      WorkspaceMembers: ['/workspaces/:workspaceId/members/:id'],
+	  WorkspaceUserRoles: ['/workspaces/:workspaceId/user_roles'],
```

Comment:
indentation looks off. mind fixing that?

Reply:
yep fixed damn editor